spec/unit/outputs/elasticsearch/http_client_spec.rb

```diff
@@ -180,6 +180,53 @@
     end
   end
 
+  describe "#bulk" do
+    subject { described_class.new(base_options) }
+
+    require "json"
+    let(:message) { "hey" }
+    let(:actions) { [
+      ["index", {:_id=>nil, :_index=>"logstash"}, {"message"=> message}],
+    ]}
+
+    context "if a message is over TARGET_BULK_BYTES" do
+      let(:target_bulk_bytes) { LogStash::Outputs::ElasticSearch::TARGET_BULK_BYTES }
+      let(:message) { "a" * (target_bulk_bytes + 1) }
+
+      it "should be handled properly" do
```

Comment:
Phrases like "should be handled properly" require that the reader have an implicit understanding of what "handled" and "properly" mean in this particular context, which doesn't effectively summarise the example in question; although I see that the existing examples aren't particularly consistent about doing so, it's also nice to use imperative language instead of "should"-type statements.

perhaps this would be better?

~~~ ruby
      it "sends the message as its own bulk payload" do
~~~